docs/build/smart-contracts/example-contracts/tokens.mdx

```diff
@@ -472,13 +473,13 @@ You have likely noticed that this example contract is broken into discrete modul
 
 For example, most of the token logic exists in the `contract.rs` module. Functions like `mint`, `burn`, `transfer`, etc. are written and programmed in that file. The Token Interface describes how some of these functions should emit events when they occur. However, keeping all that event-emitting logic bundled in with the rest of the contract code could make it harder to track what is happening in the code, and that confusion could ultimately lead to errors.
 
-Instead, we have a separate `soroban_token_sdk::TokenUtils` module that takes away all the headache of emitting events when other functions run. Here is the event emitted when a token is minted:
+Instead, we have a separate`soroban_token_sdk::TokenUtils` module that takes away all the headache of emitting events when other functions run.Here is the event emitted when a token is
```

Comment:
I'm seeing in a couple places, some find-n-replace you ran may have been a little greedy too hehe.

Reply:
Oh I see, there were times I missed a colon on a link and then prettier ran and went nuts then after the syntax was fixed prettier switched it all back.

I see, I can to a fresh checkout off the tip of main and rebase my branch back in to get a much nicer and minimal changeset.